fs-util.js

```diff
@@ -0,0 +1,11 @@
+var fs = require('fs');
```

Comment:
Im sorry to be a pain but this file is kinda redundant now right? 😇  I mean `existsSync` is jsut a re-export and isDirectory can be inline in the file.

Reply:
ok I will remove it